docs/jotai/api/utils.mdx

```diff
@@ -222,6 +222,33 @@ import deepEqual from 'fast-deep-equal'
 const fooFamily = atomFamily((param) => atom(param), deepEqual)
 ```
 
+### TypeScript
+
+Types are defined like this:
```

Comment:
Maybe change this line to:  
"You can explicitly declare the parameter, value, and atom's update function types using TypeScript generics"